pkg/k8shandler/fluentd.go

```diff
@@ -361,6 +362,21 @@ func newFluentdPodSpec(cluster *logging.ClusterLogging, proxyConfig *configv1.Pr
 		{Name: "POD_IP", ValueFrom: &v1.EnvVarSource{FieldRef: &v1.ObjectFieldSelector{APIVersion: "v1", FieldPath: "status.podIP"}}},
 	}
 
+	if cluster.Spec.Forwarder != nil {
+		if cluster.Spec.Forwarder.Fluentd.Buffer != nil {
+			if cluster.Spec.Forwarder.Fluentd.Buffer.ChunkLimitSize != "" {
+				if chunkLimitSize, err := utils.ParseQuantity(string(cluster.Spec.Forwarder.Fluentd.Buffer.ChunkLimitSize)); err == nil {
+					fluentdContainer.Env = append(fluentdContainer.Env, v1.EnvVar{Name: "BUFFER_SIZE_LIMIT", Value: strconv.FormatInt(chunkLimitSize.Value(), 10)})
+				}
+			}
+			if cluster.Spec.Forwarder.Fluentd.Buffer.TotalLimitSize != "" {
+				if totalLimitSize, err := utils.ParseQuantity(string(cluster.Spec.Forwarder.Fluentd.Buffer.TotalLimitSize)); err == nil {
+					fluentdContainer.Env = append(fluentdContainer.Env, v1.EnvVar{Name: "TOTAL_LIMIT_SIZE", Value: strconv.FormatInt(totalLimitSize.Value(), 10)})
```

Comment:
It looks like the run script doesn't take this into account.  It will require adjustments to skip calculations of the value if it is provided by the operator.  

We may also want to add some sort of warning message in the run script about exceeding disk value.  @alanconway we calculate this value now to avoid using all the disk which this change and allowed configuration risks re-introducing

Reply:
@sasagarw does this value need to be modified in this PR?

Reply:
@alanconway not compulsorily required in this PR. We can skip for now. I will just work with fluentd chunk_limit_size value.